e2e/src/main/protobuf/custom_annotation.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package com.trueaccord.proto.e2e;
+
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  import: "com.trueaccord.pb.CustomAnnotation"
+};
+
+message CustomAnnotationMessage {
+  option (scalapb.message).annotations = "@CustomAnnotation";
```

Comment:
Please add another test with 3 different annotations, to verify that the generated code works in that case too.